indigo_api/models/works.py

```diff
@@ -331,16 +337,18 @@ def points_in_time(self):
         """ Return a list of dicts describing a point in time, one entry for each date,
         in descending date order.
         """
-        amendments_expressions = self.amendments_with_initial_and_arbitrary()
+        events_expressions = self.amendments_initial_commencement_arbitrary()
         pits = []
 
-        for date, group in groupby(amendments_expressions, key=lambda x: x.date):
+        for date, group in groupby(events_expressions, key=lambda x: x.date):
             group = list(group)
             pits.append({
                 'date': date,
                 'initial': any(getattr(a, 'initial', False) for a in group),
-                'amendments': group,
-                'expressions': set(chain(*(a.expressions().all() for a in group))),
+                'amendments': [a for a in group if a._meta.object_name == 'Amendment'],
```

Comment:
Could also do `if instanceof(a, Amendment)` which is the pythonic way to test if an object has a particular class.